modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/PartitionSnapshotStorage.java

```diff
@@ -90,6 +94,7 @@ public PartitionSnapshotStorage(
             String snapshotUri,
             RaftOptions raftOptions,
             PartitionAccess partition,
+            CatalogService catalogService,
```

Comment:
Why do we have a `CatalogAccess` abstraction, if we still use `CatalogService` instead? I don't really get it